examples/advanced/urdf_loading_and_manipulation/main.py

```diff
@@ -0,0 +1,48 @@
+import blenderproc as bproc
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('urdf_file', nargs='?', default="./model.urdf", help="Path to the .urdf file")
+parser.add_argument('output_dir', nargs='?', default="examples/advanced/urdf_loading_and_manipulation/output", help="Path to where the final files will be saved")
+args = parser.parse_args()
+
+bproc.init()
+
+robot = bproc.loader.load_urdf(urdf_file=args.urdf_file)
+
+robot.hide_irrelevant_objs()
+robot.remove_link_by_index(index=0)
+robot.set_ascending_category_ids()
+
+# rotate for some parts
+robot_matrix_world = []
+for frame in range(9):
+    for link in robot.links:
+        if link.joint_type == "revolute":
+            link.set_rotation_euler(rotation_euler=0.1, mode="relative", frame=frame)
+    robot_matrix_world.append(robot.get_all_local2world_mats())
+
+# set a light source
+light = bproc.types.Light()
+light.set_type(type="POINT")
+light.set_location(location=[5, 5, 5])
+light.set_energy(energy=1000)
+
+# sample camera pose
+bproc.camera.set_intrinsics_from_blender_params(640, 480)
```

Comment:
```suggestion
bproc.camera.set_resolution(640, 480)
```
`set_intrinsics_from_blender_params` has focal length as first argument.

Reply:
sorry aber die argumente von `set_intrinsics_from_blender_params()` sind doch falsch:

https://dlr-rm.github.io/BlenderProc/blenderproc.api.camera.html#blenderproc.camera.set_intrinsics_from_blender_params

Reply:
natürlich, hab zu früh resolved!